x-pack/plugins/apm/server/routes/traces/route.ts

```diff
@@ -88,27 +89,36 @@ const tracesByIdRoute = createApmServerRoute({
     path: t.type({
       traceId: t.string,
     }),
-    query: rangeRt,
+    query: t.intersection([
+      rangeRt,
+      t.partial({ entryTransactionId: t.string }),
```

Comment:
why is `entryTransactionId ` optional? I thought it was always required?

Reply:
Yes, you're right, I'll make it required.